CONTRIBUTING.md

```diff
@@ -89,7 +89,7 @@ Manually testing Revel integration
 1. `cd` into the `examples/revelapp/` directory
 1. Run the `start-test-servers.sh` script
 1. Run `revel run`
-1. Visit `localhost:4321/handled` and verify that `payloads.json` now contains a report payload containing a handled payload, including a `session` sub-object containing 0 unhandled and 1 handled.
-1. Visit `localhost:4321/unhandled` and verify that `payloads.json` now contains an unhandled payload, including a `session` sub-object containing 1 unhandled and 0 handled.
+1. Visit `localhost:9001/handled` and verify that `payloads.json` now contains a report payload containing a handled payload, including a `session` sub-object containing 0 unhandled and 1 handled.
```

Comment:
This will need an additional step to configure the endpoints so that they point at localhost:1234